public/app/features/alerting/state/alertDef.ts

```diff
@@ -81,8 +81,14 @@ function createReducerPart(model: any) {
   return new QueryPart(model, def);
 }
 
+// state can also contain the state it was mapped from, ie. "Alerting (NoData)" which means the alert rule is "Alerting" but
+// because the alert rule was configured to use this state instead of "NoData"
+function normalizeAlertState(state: string) {
+  return state.toLowerCase().replace(/_/g, '').split(' ')[0];
+}
+
 function getStateDisplayModel(state: string) {
-  const normalizedState = state.toLowerCase().replace(/_/g, '');
```

Comment:
Why would this cause a crash? Should the code that is crashing have some defenses added to it?

Reply:
It's down on 163? Right? I don't see why we should `throw` there instead of just using some ugly text field there.